src/arviz_plots/plots/forestplot.py

```diff
@@ -131,40 +131,27 @@ def plot_forest(
 
     Examples
     --------
-    The following examples focus on behaviour specific to ``plot_forest``.
+    The following examples focus on behaviour specific to ``plot_forest`` and aim to illustrate
+    usage of specific arguments.
```

Comment:
```suggestion
```

Reply:
I am not sure this intro is that useful so I suggest remove most of them and move the first sentence after the example and before the mini-gallery

Reply:
maybe we could get rid of that completely and link to plots_intro in a See Also section (which we should add at some point, similarly to the ones in current arviz)

Reply:
that also makes sense